### DISCUSSION TITLE
AWS Toolkit

### DISCUSSION THREAD

**Discussion by kurt-rhee** [zed-industries/zed#23875] [Feature Requests]
### Check for existing issues

- [X] Completed

### Describe the feature

I'd like to have AWS Toolkit so I can develop locally while authenticating through the AWS IAM identity center.

### Environment

Zed: v0.159.10 (Zed)
OS: macOS 15.0.1
Memory: 16 GiB
Architecture: aarch64

### If applicable, add mockups / screenshots to help present your vision of the feature

![image](https://github.com/user-attachments/assets/7ace158c-42e9-4a6c-a50a-5e82c8428b6d)

**Comment by bfoley12** [comment]
Also interested in this feature. Has there been any movement on it?

**Reply by neezymakeitez** [reply]
I can make one
